### PR TITLE
Feat 262 remove d6t rows

### LINKED ISSUE #260: Decrease size of quantity field in D6T table
**Is your feature request related to a problem? Please describe.**
The size of the quantity field in the Enter Receipt table is larger than realistic order sizes. The extensive width takes up screen real-estate.  

**Describe the solution you'd like**
Change the size of the quantity field. Consider setting a smaller max-width or making the column a fixed width.

**Describe alternatives you've considered**
N/A

**Additional context**
The column with the quantity field needs to be smaller.
image

### LINKED ISSUE #262: Add a remove row button to each row created in the D6T table
**Is your feature request related to a problem? Please describe.**
Users can add multiple rows to the D6T table, but can not remove individual rows. Currently the only way to remove a row is to refresh the page and start searching for SDNs from scratch. 

**Describe the solution you'd like**
Add a button to every row in the D6T table that allows user to remove the row. 

**Describe alternatives you've considered**
Alternatively, we could remove the users ability to submit for more than one SDN at a time.

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by scottpiercewDDS**
Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist
I have…
- [x] summarized below my changes and noted which issues (if any) this pull request fixes or addresses.
- [x] thoroughly outlined below the steps necessary to test my changes.
- [x] attached screenshots illustrating relevant behavior before and after my changes.
- ~~[ ] review and update SSAG documentation if needed.~~

## Summary of Changes
- Closes #262 
- Closes #260 
- The size of the quantity field in the D6T table has been reduced to take up less screen space. Users can now remove individual rows from the D6T table by selecting the remove X on the left side of each row.

## Testing
To verify the changes proposed in this pull request…
1. run frontend unit tests

## Screenshots
image

### CODE REVIEW

File: frontend/src/pages/EnterReceiptPage/__tests__/enterReceiptPage.spec.tsx [hunk 1]
```diff
@@ -57,6 +57,31 @@ describe("EnterReceiptPage Component", () => {
...
+    const inputField = getByPlaceholderText("SDN");
+    const submit = getByText("Search");
+    fireEvent.change(inputField, {
+      target: { value: "wrongsdn" }
+    });
+    await wait(() => {
+      expect(inputField).toHaveValue("wrongsdn");
+    });
+    fireEvent.click(submit);
+    const removeIcon = container.querySelector("button.usa-button") as Element;
```
Comment (jhackshaw-dds) [inline frontend/src/pages/EnterReceiptPage/__tests__/enterReceiptPage.spec.tsx]: after clicking submit, can you
```
await wait(() => {
  expect(getByText('wrongsdn')).toBeIntheDocument()
})
```

File: frontend/src/pages/EnterReceiptPage/__tests__/enterReceiptPage.spec.tsx [hunk 2]
```diff
@@ -57,6 +57,31 @@ describe("EnterReceiptPage Component", () => {
...
+    fireEvent.click(removeIcon);
+    await wait(() => {
+      expect(removeIcon).not.toBeInTheDocument();
+    });
```
Comment (jhackshaw-dds) [inline frontend/src/pages/EnterReceiptPage/__tests__/enterReceiptPage.spec.tsx]: and then this assertion could expect the sdn to no longer be in the document

`expect(queryByText('wrongsdn')).not.toBeInTheDocument()`

File: frontend/src/solo-types/index.ts [hunk 3]
```diff
@@ -134,3 +134,18 @@ export interface PaginatedApiResponse<T> {
...
+{
+  results: [
+    Doc, Doc, Doc x25
+  ],
+  count: 3560
+}
+
+3560 // 25
+
+*/
```
Comment (jhackshaw-dds) [inline frontend/src/solo-types/index.ts]: can you remove this

**Review by jhackshaw-dds — CHANGES_REQUESTED** [review]
Looks good, just some minor changes for testing and some left over comments.

File: frontend/src/pages/EnterReceiptPage/__tests__/enterReceiptPage.spec.tsx (hunk 2 above)
Comment (scottpiercewDDS) [reply frontend/src/pages/EnterReceiptPage/__tests__/enterReceiptPage.spec.tsx]: Is it better to use both assertions or replace with the new assertion?

**Review by jhackshaw-dds — APPROVED** [review]

**Pull request merged**